required/latex-lab/latex-lab-table.dtx

```diff
@@ -546,20 +546,34 @@
 % \enquote{vmode} tables like longtable.
 %
 % \begin{plugdecl}{Table}
+%  Inside a TD or TR Part is not allowed as child. For structures that restore
+%  paragraph settings we therefore need a special plug that adjust the settings.
+%  Currently we set the para main tag to Div. This leads to double Div-structures
+%  but flattening the paragraph doesn't work, it errors if there is a list inside. 
+%    \begin{macrocode}
+\NewSocketPlug{tagsupport/para/restore}{Table}
+  {
+    \tl_set:Nn    \l__tag_para_main_tag_tl {Div}
+    \tl_set_eq:NN \l__tag_para_tag_tl\l__tag_para_tag_default_tl
+    \bool_set_true:N \l__tag_para_bool
+    \bool_set_false:N \l__tag_para_flattened_bool
+  }
+%    \end{macrocode}
+% \end{plugdecl}
+% 
+% \begin{plugdecl}{Table}
 %    Inside a table we currently only disable paratagging. We assume
 %    that these sockets are in an environment group, so there is no
```

Comment:
called in a group   

may not be an environment. I guess \parbox owuld have the same problem, wouldn't it?

Reply:
well what was meant was that the table socket is called inside a tabular environment and so we don't have to reset at \end{tabular}. But I changed to "called in a group".